client/src/components/RegisterPage/Registration.js

```diff
@@ -26,47 +34,91 @@ const Register = (props) => {
         password: user["password"],
         secondPass: user["secondPass"],
       });
+      setUserError(false);
     } else if (name === "email") {
       setUser({
         username: user["username"],
         email: value,
         password: user["password"],
         secondPass: user["secondPass"],
       });
+      setEmailError(false);
     } else if (name === "password") {
       setUser({
         username: user["username"],
         email: user["email"],
         password: value,
         secondPass: user["secondPass"],
       });
+      setPasswordError(false);
     } else if (name === "secondPass") {
       setUser({
         username: user["username"],
         email: user["email"],
         password: user["password"],
         secondPass: value,
       });
+      setPasswordError(false);
     }
   };
 
-  function submitForm() {
+  const submitForm = async () => {
     // Place stuff in here to add person to database
-    console.log("Submitting");
+    var validUser = validateUser();
+    var validEmail = validateEmail();
+    var validPassword = validatePassword();
     if (
-      validateUser() === 0 &&
-      validateEmail() === 0 &&
-      validatePassword() === 0
+      validUser === 0 &&
+      validEmail === 0 &&
+      validPassword === 0
     ) {
-      console.log("All good to add them to database and send them off");
-      window.location.href = "/browse";
+      let resp = getUser(user.email, user.password);
+      console.log("resp value: " + resp);
+      // resp.status == 200 means something was there
+      // resp.status == 401 means the credentials were not taken
+      if (resp.status === 200) {
+        console.log("Invalid credentials");
+        setError(true);
+      }
+      else if (resp === 401) {
+        await axios.post("https://127.0.0.1/users", {
+          auth: {
```

Comment:
Payload for registering is just 
{
username,
email,
password
} no need for auth.